.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create an issue about a bug on the devdocs website
+---
+
+# Bug report
+
+## Description
+
+<!-- (REQUIRED) What is the issue/current behavior? -->
+
+### Steps to reproduce
+
+<!-- (OPTIONAL) What needs to be done to replicate this issue? (provide Gist if needed) -->
+
+1. First Step
+2. Second Step
+3. Etc.
+
+## Expected result
+
+<!-- (REQUIRED) When this issue is resolved, what is the expected result/behavior? -->
```

Comment:
Not sure that an Expected result is needed for a bug report because the negation of the description pretty much gives you the expected result.

Reply:
It depends on how well and thought out the description is written. By requiring this section, we have something to point to and ask, "Does this solution fulfill what is described in the 'Expected result' section?"